src/adhocracy_mercator/adhocracy_mercator/resources/mercator2.py

```diff
@@ -0,0 +1,155 @@
+"""Mercator 2 proposal."""
+
```

Comment:
"Mercator 2" is not very descriptive. We should explain what is the difference to "mercator", like  "Proposal forms for the  2016 participation process."